CruzHacks/cruzhacks-2021-website
# MLH Badge, Favicon, and new Scaffolding

Problem
=======
As an organization, 
i want to see the MLH Banner on our site 
So we can be MLH compliant 

As a developer, 
I want the repo to be scaffolded before I start developing
So I can focus on my component

[Link to MLH Banner Ticket](https://app.asana.com/0/1197738625416321/1198181676235769/f)
[Link to Scaffolding Ticket](https://app.asana.com/0/1197738625416321/1199382847026325/f)

Solution
========
Copied the link from [MLH website](https://mlh.io/brand-guidelines#trust-badge) and created a component for the MLH banner, which we will need to import in every *view* that we have. 

To begin reflecting some changes for our new site, I have also added the favicon that the design team had for our site! Additionally, we have create a new scaffolding for our site going forward.

```bash 
./src
├── components
│   ├── Button
│   ├── EmailSubscription
│   ├── MLHBanner
│   ├── Navbar
│   └── PostcardStack
├── fonts
├── images
│   └── postcards
├── routes
├── stylesheets
└── views
    ├── Hero
    ├── Home
    │   └── components
    │       ├── FAQ
    │       ├── Milestones
    │       ├── Mission
    │       ├── Prizes
    │       ├── Schedule
    │       ├── Speakers
    │       └── Sponsors
    └── Team
```

Change summary:
---------------
* added badge & new favicon 89947df
* added scaffolding for component 5d68b00

Steps to Verify:
----------------
1. Checkout branch & start local dev server
1. Verify that badge appears in main view, but disappears when it's in mobile view
1. Verify that new favicon is present

Screenshots (optional):
-----------------------
**Favicon**
<img width="216" alt="Screen Shot 2020-11-28 at 4 40 25 PM" src="https://user-images.githubusercontent.com/40477553/100529077-6c7f3c80-3198-11eb-8138-1d195c14ac3d.png">

**MLH Banner**
<img width="1440" alt="Screen Shot 2020-11-28 at 4 40 44 PM" src="https://user-images.githubusercontent.com/40477553/100529082-7739d180-3198-11eb-959b-e9a1a5420663.png">
